Application/Commands/CreateBookCommandTests.cs

```diff
@@ -68,16 +75,43 @@ public async Task CreateAsync_ShouldAddNewBookToDbSet()
   }
 
   [Fact]
-  public async Task CreateWithouAuthorsAsync_ShouldThrowException()
+  public async Task CreateWithoutAuthorsAsync_ShouldThrowException()
   {
     var createBookRequest = new CreateBookCommandParams
     {
       Title = "Test Book",
       Annotation = "Test annotation",
       Language = 0,
+      KnowledgeAreas = new List<KnowledgeArea>
+      {
+        new KnowledgeArea { 
+          Id = 2, 
+          Name = "Backend"
+        }
+      },
       CoverUrl = "http://test-images.com/img404.png",
       Authors = new List<Author> { }
     };
+    
+    await Assert.ThrowsAsync<ArgumentException>(async () => await _command.CreateAsync(createBookRequest, TENANT_ID));
+  }
+
+  [Fact]
+  public async Task CreateWithoutKnowledgeAreaAsync_ShouldThrowException()
+  {
+    var createBookRequest = new CreateBookCommandParams
+    {
+      Title = "Test Book",
+      Annotation = "Test annotation",
+      Language = 0,
+      KnowledgeAreas = new List<KnowledgeArea>(),
+      CoverUrl = "http://test-images.com/img404.png",
+      Authors = new List<Author>
+      {
+        new Author { FullName = "Test Author" }
+      },
+      CountOfCopies = 1
+    };
 
     await Assert.ThrowsAsync<ArgumentException>(async () => await _command.CreateAsync(createBookRequest, TENANT_ID));
```

Comment:
Let's check the message since there are several ArgumentException that can be thrown.